tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -550,6 +550,16 @@ platform_tests/fwutil/test_fwutil.py::test_fwutil_auto:
   skip:
     reason: "Command not yet merged into sonic-utilites"
 
+#######################################
+#####  test_memory_exhaustion.py  #####
+#######################################
+platform_tests/test_memory_exhaustion.py:
+  skip:
+    reason: "Skip this test case until vendors fix the issue"
+    conditions:
+      - "hwsku in ['Celestica-DX010-C32', 'Celestica-E1031-T48S4']"
+      - https://github.com/Azure/sonic-buildimage/issues/10339
```

Comment:
Do we support put an external repo issue as condition here?

Reply:
Yes, the conditional mark plugin supports querying any Github issue.